plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -521,6 +521,8 @@ public void testHiveViewsCannotBeAccessed()
         String viewName = "dummy_view";
         hiveHadoop.runOnHive("CREATE DATABASE " + schemaName);
         hiveHadoop.runOnHive(format("CREATE VIEW %s.%s AS SELECT * FROM %s.customer", schemaName, viewName, SCHEMA));
+        assertUpdate("CREATE SCHEMA dummy"); // Force database cache invalidation
+        assertUpdate("DROP SCHEMA dummy");
```

Comment:
What happens if other tests populate the cache between L525 and the following assertions?

Reply:
> What happens if other tests populate the cache between L525 and the following assertions, or do you expect such a situation never happens?
That should not be a problem since problem occur only when there is no cache update. All cache updates after direct hive calls will see newly created database. But I will look into other options as they may be more clean solutions.